src/sqlfluff/dialects/dialect_mysql.py

```diff
@@ -380,14 +381,209 @@ class CreateTableStatementSegment(ansi.CreateTableStatementSegment):
     https://dev.mysql.com/doc/refman/8.0/en/create-table.html
     """
 
+    ES = Ref("ExpressionSegment")
```

Comment:
I don't love this aliasing here. There's nothing inherently wrong with the pattern. But we don't use it elsewhere, so I think it overall makes it less readable.

Reply:
I'm not a big fan either; it was the only way I was able to get the linter to accept the code due to column lengths without it.

Reply:
I agree, but in any of my limited attempts to simply break the sequences out into their own segments, I end up getting test failures. I'm pretty limited on time right now, but I'm not opposed to getting rid of the partition/subpartition options. It was just a lacking feature I thought I'd implement while I was at it, but I don't really use it personally, so I don't mind stripping it to get this PR pushed through. Lmk your thoughts